### DISCUSSION TITLE
LTSC EASY Install & for those who dislike MS store ??

### DISCUSSION THREAD

**Discussion by methanoid** [microsoft/winget-cli#2828] [Ideas]
I cannot believe how unnecessarily complicated the install process is if you use Windows LTSC or (violently) dislike the idea of the MS Store. Why not a simple single file EXE/MSI installer? Simple installation should be the basis of any project. I'd hoped the Winget-gui project would include the CLI install in it but unfortunately not.

Chocolatey is install with a single line Powershell script (and doesnt need MS store). Why can't Winget do this (at LEAST)???

**Comment by Masamune3210** [comment]
![image](https://user-images.githubusercontent.com/1053504/211156233-367c0a0b-8e36-4113-aeb9-27f02ab6cfab.png)
Unless LTSC lacks support for MSIX packages, there is already a way to install winget without using the Store.

**Comment by methanoid** [comment]
I stand by the earlier comment about an easy single file installer for all Windows inc LTSC but I managed to get installed relatively easily and make a PS script to simplify future installs.

But it could/should be easier IMHO
